bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/DatetimePicker.java

```diff
@@ -0,0 +1,99 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.form.datetime;
+
+import org.apache.wicket.extensions.markup.html.form.DateTextField;
+import org.apache.wicket.model.IModel;
+
+import java.util.Date;
+
+/**
+ * DateTime input field with a Eonasdan datetimepicker plugin.
+ *
+ * @author Alexey Volkov
+ * @since 01.02.2015
+ */
+public class DatetimePicker extends DateTextField {
+
+    private static final long serialVersionUID = -9043590922065213472L;
+    private DatetimePickerConfig config = newDefaultConfig();
```

Comment:
Better pass the config to the constructor.
It is a bad practice to call overridable method in the constructor.